src/sources/gcp_pubsub.rs

```diff
@@ -324,10 +324,9 @@ impl PubsubSource {
         response: proto::StreamingPullResponse,
         finalizer: &Option<Finalizer>,
     ) {
-        emit!(HttpClientBytesReceived {
+        emit!(BytesReceived {
             byte_size: response.size_of(),
             protocol: self.uri.scheme().map(Scheme::as_str).unwrap_or("http"),
-            endpoint: &self.endpoint,
```

Comment:
Ditto question about dropping the `endpoint` tag here.

Reply:
Same thing.  The endpoint is fixed for the source -- via the config -- so the metric is differentiable by the component ID alone, and there's no variability here to actually push into the logs/metrics.